src/command_modules/azure-cli-network/azure/cli/command_modules/network/commands.py

```diff
@@ -7,7 +7,7 @@
 
 from azure.cli.core.commands.arm import cli_generic_update_command, cli_generic_wait_command
 from azure.cli.core.commands import DeploymentOutputLongRunningOperation, cli_command
-from ._client_factory import * #pylint: disable=wildcard-import
+from ._client_factory import * # pylint: disable=wildcard-import, unused-wildcard-import
```

Comment:
Ah thought pylint as say _all_ of the imports weren't used rather than just one of them.